### PR TITLE
refactor: sport module migrate (#684)

### PR DESCRIPTION
**Pull request by 00thirdeye00**
sport module migration from class to functions within sport namespace

### CODE REVIEW

**Comment by 00thirdeye00** [comment]
@cieslarmichal Please check out the discord general channel for this merge :)

File: include/faker-cxx/Sport.h [hunk 1]
```diff
@@ -2,11 +2,8 @@
 
 #include <string_view>
 
-namespace faker
+namespace faker::Sport
```
Comment (cieslarmichal) [inline include/faker-cxx/Sport.h:5]: Please use lowercase for namespaces as in other modules

**Review by cieslarmichal — CHANGES_REQUESTED** [review]

**Review by cieslarmichal — CHANGES_REQUESTED** [review]
Two things:

1. Add namespace to data file as well
2. Write a line in CHANGELOG.md that sport function was renamed to sportName please

File: include/faker-cxx/Sport.h (hunk 1 above)
Comment (00thirdeye00) [reply include/faker-cxx/Sport.h]: modified namespaces to lowecase

**Review by cieslarmichal — APPROVED** [review]

**Pull request merged**